docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
```

Comment:
Maybe start with a sentence that effectively says "if you want to update this documentation, here's how" so it's obvious that this is documentation for the documentation and not documentation for building and using ZMK?